dt-assets/js/comments.js

```diff
@@ -359,6 +387,62 @@ jQuery(document).ready(function($) {
         activity: array
       }))
     }
+    document.querySelectorAll('.reactions__dropdown').forEach((element) => {
+      const commentId = element.dataset.commentId
+      const emojis = emojiButtons()
+      const reactionForm = document.createElement('form')
+      reactionForm.classList.add('pick-reaction-form')
+      reactionForm.innerHTML = `
+        ${emojis}
+      `
+      reactionForm.addEventListener('submit', (e) => {
+        e.preventDefault()
+        const userId = commentsSettings.current_user_id
+        const reaction = e.submitter.value
+        rest_api.toggle_comment_reaction(postType, postId, commentId, userId, reaction)
+      })
+      element.appendChild(reactionForm)
+    })
+    document.querySelectorAll('#comments-wrapper [data-toggle]').forEach((element) => {
+      const dropdownId = $(element).data('toggle')
+      const dropdownElement = document.querySelector(`#${dropdownId}`)
+      element.addEventListener('click', (e) => {
+        const style = getComputedStyle(dropdownElement)
+        element.toggleAttribute('open')
+        if (style.visibility === 'hidden') {
+          dropdownElement.style.visibility = 'visible'
+          dropdownElement.style.display = 'block'
+        } else {
+          dropdownElement.style.visibility = 'hidden'
+          dropdownElement.style.display = 'none'
+        }
+      })
+    })
+    document.querySelectorAll('.comment-reaction').forEach((element) => {
+      element.addEventListener('click', (e) => {
+        const commentId = e.target.dataset.commentId
+        const userId = commentsSettings.current_user_id
+        const reaction = e.target.dataset.reactionValue
+        rest_api.toggle_comment_reaction(postType, postId, commentId, userId, reaction)
+      })
+    })
+  }
```

Comment:
This uses vanilla javascript (which I prefer), but we are using jquery throughout the project. I personally don't think it is worth changing and the vanilla JS approach is better moving forward, but we should have @corsacca weigh in on if we should use jquery functions here for consistency with the rest of the project.

Reply:
plain JS is totally cool :)